### PR TITLE
Upgraded from OpenShift 4.9 to 4.10

### PR DESCRIPTION
**Pull request by eric-nadeau**
## Change Overview

1. Bumped up various instances of 4.9/4_9/4-9 to 4.10/4_10/4-10
   1. In these files:
      1. Makefile
      2. build/integration-test.sh
      3. pkg/app/utils.go
      4. pkg/testing/integration_register.go
   2. Based on these PRs to upgrade from 4.7 to 4.9:
      1. https://github.com/kanisterio/kanister/pull/1181/files
      2. https://github.com/kanisterio/kanister/pull/1191/files
   3. One exception was PostgreSQL's namespace
      1. Changed from postgresdepconf4-5-test to postgresdepconf4-10-test
      2. Seems we missed it in previous upgrades

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [X] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues

- #1474

## Test Plan

- [X] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

```
1. Disabled minio check

   1. Ensured there were no local changes

      veeam:~/src/kanister$ git status
      On branch 1474_SupportOpenShift4.10
      Your branch is up to date with 'origin/1474_SupportOpenShift4.10'.
      
      nothing to commit, working tree clean

   2. Patched integration-test.sh

      veeam:~/src/kanister$ sed -i '' -e '/check_dependencies$/d' build/integration-test.sh

   3. Inspected the changes
      
      veeam:~/src/kanister$ git diff
      diff --git a/build/integration-test.sh b/build/integration-test.sh
      index e717318d..10827f81 100755
      --- a/build/integration-test.sh
      +++ b/build/integration-test.sh
      @@ -100,7 +100,6 @@ case "${1}" in
	       ;;
       esac

      -check_dependencies
       echo "Running integration tests:"
       pushd ${INTEGRATION_TEST_DIR}
       go test -v ${TEST_OPTIONS} -check.f "${TEST_APPS}" -installsuffix "static" . -check.v

2. Ran openshift-test
   
   1. Deployed an OpenShift 4.10 cluster
   2. Pointed ~/.kube/config to it
   3. Verified the cluster is running 4.10

      veeam:~/src/kanister$ oc version | grep Server
      Server Version: 4.10.17

   4. Ensured my environment variables were set

      veeam:~/src/kanister$ printenv | awk -F= '/AWS/{print$1}'
      AWS_ACCESS_KEY_ID
      AWS_SECRET_ACCESS_KEY
      AWS_REGION

   5. Ran openshift tests

      veeam:~/src/kanister$ /usr/bin/time -p make openshift-test
      Running integration tests:
      ~/src/kanister/pkg/testing ~/src/kanister
      === RUN   Test

      ****************************************
      *** truncated 114 lines of JSON logs ***
      ****************************************

      OK: 2 passed
      --- PASS: Test (109.93s)
      PASS
      ok  	github.com/kanisterio/kanister/pkg/testing	110.509s
      ~/src/kanister
      real 115.21
      user 6.37
      sys 4.25
      veeam:~/src/kanister$ echo $?
      0

3. Ran integration-test

   1. Deployed a plain cluster in AWS' EKS

   2. Pointed ~/.kube/config to it
      
   3. Ensured my environment variables were set

      veeam:~/src/kanister$ printenv | awk -F= '/AWS/{print$1}'
      AWS_ACCESS_KEY_ID
      AWS_SECRET_ACCESS_KEY
      AWS_REGION

   4. Called the make target to run integration tests

      veeam:~/src/kanister$ /usr/bin/time -p make integration-test DOCKER_BUILD=false
      Running integration tests:
      ~/src/kanister/pkg/testing ~/src/kanister
      === RUN   Test

      ****************************************
      *** truncated 355 lines of JSON logs ***
      ****************************************

      OK: 6 passed
      --- PASS: Test (231.88s)
      PASS
      ok  	github.com/kanisterio/kanister/pkg/testing	232.463s
      ~/src/kanister
      real 236.56
      user 18.67
      sys 7.75
      veeam:~/src/kanister$ echo $?
      0

```

### CODE REVIEW

**Review by tdmanv — APPROVED** [review]

File: build/integration-test.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ Where app-type is one of [short|all]:
   all: Runs e2e integration tests for all apps
   OR
   You can also provide regex to match apps you want to run.
-  openshift ocp_version=<ocp_version>: Runs e2e integration tests for specific version of OpenShift apps, OCP version can be provided using ocp_version argument. Currently supported versions are 3.11, 4.4 and 4.5.
+  openshift ocp_version=<ocp_version>: Runs e2e integration tests for specific version of OpenShift apps, OCP version can be provided using ocp_version argument. Currently supported versions are 3.11, 4.4, 4.5 and 4.10.
```
Comment (pavannd1) [inline build/integration-test.sh:59]: Same as above

File: build/integration-test.sh [hunk 2]
```diff
@@ -32,7 +32,7 @@ OC_APPS4_4="MysqlDBDepConfig4_4|MongoDBDepConfig4_4|PostgreSQLDepConfig4_4"
 OC_APPS4_5="MysqlDBDepConfig4_5|MongoDBDepConfig4_5|PostgreSQLDepConfig4_5"
```
Comment (pavannd1) [inline build/integration-test.sh:32]: Should we just remove older versions? I'm confident that we don't support 3.11 anymore but we can reduce the tests to just the latest version

**Review by pavannd1 — APPROVED** [review]
LGTM

Was thinking about removing older versions. We can address that later 👍🏼

File: build/integration-test.sh (hunk 2 above)
Comment (eric-nadeau) [reply build/integration-test.sh:32]: From the way the code is written it looked like we wanted to support some specific legacy versions in addition to the latest. This ticket was opened to track this idea: https://github.com/kanisterio/kanister/issues/1512

**Pull request merged**